container-images/scripts/build_llama_and_whisper.sh

```diff
@@ -253,14 +253,13 @@ clone_and_build_ramalama() {
   git clone https://github.com/containers/ramalama
   cd ramalama
   git submodule update --init --recursive
-  pip install . --prefix=/usr
+  python3 -m pip install . --prefix="$1"
   cd ..
   rm -rf ramalama
 }
 
 build_rag() {
-    python3 -m pip install qdrant_client fastembed openai fastapi uvicorn
-    rag_framework load
```

Comment:
What does "rag_framework load" actually do?

I fixed this another why, by copying rag_framework into the container image

Reply:
It pulls images required for doc2rag, so it should only be called in the -rag containerfiles, not in all containerfiles.